CDCgov/prime-reportstream
# 1420 Verify Coded Element codes in HL7 ingest

This PR adds the verification of coded element codes (e.g. hl70189 H for Hispanic).

To test:
1. Run `gradle test testIntegration` and verify all tests pass
2. Run `./prime test` and verify all tests pass
3. Run `./prime test --run compare` and verify all test pass

## Changes
- Validate CODE fields against existing codes and add warning if unable to validate
- Added CWE code to hl70125
- Added 119297000 code to specimen type.  This came from a real life example we have
- Fixed issue with CSV compare not handling new schema fields
- Updated test files to reflect new code

## Checklist

### Testing

### Process

## Fixes
- #1420

## To Be Done
